tools/langs/cplusplus_vrv.py

```diff
@@ -661,29 +689,31 @@ def __create_att_classes(schema, outdir, includes_dir):
             checkers = ""
             setters += SETTERS_IMPL_TEMPLATE_GRP_START.format(**{
                                                               "attGroupNameUpper": schema.cc(schema.strpatt(gp)),
-                                                              "attId": "ATT_{0}".format(schema.cc(schema.strpatt(gp)).upper()) })
+                                                              "attId": "ATT_{0}".format(schema.cc(schema.strpatt(gp)).upper())})
             getters += GETTERS_IMPL_TEMPLATE_GRP_START.format(**{
                                                               "attGroupNameUpper": schema.cc(schema.strpatt(gp)),
-                                                              "attId": "ATT_{0}".format(schema.cc(schema.strpatt(gp)).upper()) })
+                                                              "attId": "ATT_{0}".format(schema.cc(schema.strpatt(gp)).upper())})
 
             for att in atts:
                 if len(att.split("|")) > 1:
                     # we have a namespaced attribute
-                    ns,att = att.split("|")
+                    ns, att = att.split("|")
                     prefix = NS_PREFIX_MAP[ns]
-                    #nssubstr = {
+                    # nssubstr = {
                     #    "prefix": NS_PREFIX_MAP[ns],
                     #    "href": ns
-                    #}
+                    # }
                     #nsDef = NAMESPACE_TEMPLATE.format(**nssubstr)
                     attrNs = "s, "
                 else:
                     prefix = ""
                     nsDef = ""
                     attrNs = ""
-                atttype, atttypename = vrv_getatttype(schema.schema, module, gp, att, includes_dir)
-                attdefault, atttypename, converters = vrv_getattdefault(schema.schema, module, gp, att, includes_dir)
-                
+                atttype, atttypename = vrv_getatttype(
```

Comment:
I really find the breaking of function calls over multiple lines annoying to read if they're not obnoxiously long. Adhering a hard border of 80 characters seems to be detrimental to readability here.